torch/_dynamo/__init__.py

```diff
@@ -4,14 +4,14 @@
 from .decorators import assume_constant_result, disable, run
 from .eval_frame import (
     explain,
-    export,
     is_dynamo_supported,
     optimize,
     optimize_assert,
     OptimizedModule,
     reset_code,
 )
 from .exc import IncorrectUsage
+from .export import export
```

Comment:
My loudest "let's fucking go"

Also, to anyone reading this, I made him do it.